log.go

```diff
@@ -21,6 +21,8 @@ const (
 	RFC5424Log = "<%d>%d %s %s %s %d ID%d %s %s"
 	// CommonLogFormat : {host} {user-identifier} {auth-user-id} [{datetime}] "{method} {request} {protocol}" {response-code} {bytes}
 	CommonLogFormat = "%s - %s [%s] \"%s %s %s\" %d %d"
+	// JSONLogFormat : {"host": "{host}", "user-identifier": "{user-identifier}", "datetime": "{datetime}", "method": "{method}", "request": "{request}", "protocol": "{protocol}", "status", {status}, "bytes": {bytes}, "referrer": "{referrer}"}
+	JSONLogFormat = `{"host":"%s", "user-identifier":"%s", "datetime":"%s", "method": "%s", "request": "%s", "protocol":"%s", "status":%d, "bytes":%d, "referrer": "%s"}`
```

Comment:
`referer` is correct for this referrer..!

Here is a reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer

Reply:
Sorry I don't understand what you mean.

Reply:
There is a `referrer` field in there now with a URL? Is that not what you were requesting?

Reply:
For the historical reasons, the referrer of HTTP is spelled as `referer`, not `referrer`. 

You can find the details in https://en.wikipedia.org/wiki/HTTP_referer.

> The HTTP referer (a misspelling of referrer[1]) is an optional HTTP header field that identifies the address of the webpage (i.e. the URI or IRI) which is linked to the resource being requested. By checking the referrer, the new webpage can see where the request originated.

Reply:
Ah gotcha. Done.